src/ApplicationCore/Entities/FraudProtectionApiModels/SignInHistory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Contoso.FraudProtection.ApplicationCore.Entities.FraudProtectionApiModels
+{
+    public class SignInHistory
+    {
+        public string SignUpIpAddress { get; set; }
```

Comment:
For this property and all properties in the sign in models, can you please add comments?